### DISCUSSION TITLE
Where is loadCurrentSettings?

### DISCUSSION THREAD

**Discussion by mp035** [RaspAP/raspap-webgui#748] [General]
Hi @billz,
I am refactoring the last of the templates into views, and I am getting a client side error "Uncaught ReferenceError: loadCurrentSettings is not defined"

Referenced on line 47 of app/js/custom.js

It looks like the function is included in custom.min.js, but has been removed since the last minification.  I'm not sure whether to remove the reference to it, or try to restore the function body from git history.  Could you advise?

Thanks.

**Comment by mp035** [comment]
Wait, I just found it.  I can see you removed it in commit a21009e049861c565df948a0436d9d08d5bdb701.  I've removed the reference to it in custom.js in my branch.

**Reply by billz** [reply]
Correct, this is deprecated with the 2.6-beta release.
